bitnami/tensorflow-inception/templates/deployment.yaml

```diff
@@ -43,16 +43,7 @@ spec:
         volumeMounts:
         - name: seed
           mountPath: "/bitnami/model-data"
-        - name: data
-          mountPath: /bitnami/tensorflow-serving
       volumes:
       - name: seed
         persistentVolumeClaim:
-          claimName: {{ template "fullname" . }}-seed-inception
-      - name: data
-      {{- if .Values.persistence.enabled }}
-        persistentVolumeClaim:
-          claimName: {{ template "fullname" . }}
-      {{- else }}
-        emptyDir: {}
-      {{- end -}}
+          claimName: {{ template "fullname" . }}-seed-inception
```

Comment:
Are you requesting a claim on every scenario? That's not what we did on other apps

Reply:
👍